package oops.exceptionall;

public class UncheckedException {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
System.out.println(10/0);
	}

}
//When we run this file we got exception becouse compiler is not forcing to handle exception but here one exception will 
//occur java.lang.ArithmeticException show it is runtime exception